generators/typescript/utils/core-utilities/reflect/__test__/setObjectProperty.test.ts

```diff
@@ -0,0 +1,74 @@
+import { setObjectProperty } from "../setObjectProperty";
+
+interface TestCase {
+    description: string;
+    giveObject: object;
+    givePath: string;
+    giveValue: any;
+    wantObject: object;
+}
+
+describe("Test setObjectProperty", () => {
```

Comment:
@amckinney i think `reflect` is probably a little weird in TS (not going to be easy for someone poking in to understand. 

Do we have a `utils` folder in core? I could see this just going in there as one of the helper functions ? or a `helpers` folder?

Reply:
Sounds good, we don't have a good place for it already, but happy to name it something more generic like `utils`.